packages/core/src/logger/loggerOptions.ts

```diff
@@ -0,0 +1,15 @@
+import { Level, LoggerInterface } from '../coreInterfaces';
+import { LoggingConfiguration } from './loggerConfiguration';
+
+export interface LoggingOptions {
+  logger?: LoggerInterface;
+  loggingConfig?: LoggingConfiguration;
+  enabled: boolean;
+}
+
+export class ConsoleLogger implements LoggerInterface {
+  public log(_level: Level, message: string, ...params: any[]): void {
+    // tslint:disable-next-line:no-console
+    console.log(message, params);
```

Comment:
Can you share some examples showing how the output for this in the PR conversation thread?

Note that we must also include the level in the message that is logged to the console.

Reply:
![image](https://github.com/apimatic/apimatic-js-runtime/assets/109739087/4f8b0e50-5c6d-4db8-bba4-77eeb4914757)